static/src/js/components/ErrorBoundary/ErrorBoundary.jsx

```diff
@@ -17,9 +17,18 @@ class ErrorBoundary extends React.Component {
   }
 
   static getDerivedStateFromError(error) {
+    let errorMessage = error.message
```

Comment:
You should add a test that mocks this response

Reply:
Added in ErrorBoundary.test.js